src/aks-preview/HISTORY.rst

```diff
@@ -12,8 +12,14 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+0.5.77
```

Comment:
Please also update the `VERSION` in `setup.py` to `0.5.77` if you do want to release a new version.

Reply:
Done